### PR TITLE
Issue 19 veronika ponizova

### PR DESCRIPTION
**Pull request by dyupon**

### CODE REVIEW

File: tasks [hunk 1]
```diff
@@ -12,7 +12,7 @@
 33 + 66
 71 + 44
 14 + 35
-83 - 34
+83 - 34 = 0
```
Comment (OLavrik) [inline tasks]: Mistake , please correct it)

File: tasks [hunk 2]
```diff
@@ -1,7 +1,7 @@
-55 - 31
+55 - 3
```
Comment (OLavrik) [inline tasks]: In tasks we have used "double" numbers, please change)

**Comment by dyupon** [comment]
@OLavrik 
please review once again

**Comment by dyupon** [comment]
pull request seems to have incorrect format

**Pull request closed without merging**